apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx

```diff
@@ -0,0 +1,161 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+export const COLUMNS = {
+  NAME: 0,
+  NUM_MULTIPLE_CHOICE_CORRECT: 1,
+  NUM_MULTIPLE_CHOICE: 2,
+  PERCENT_CORRECT: 3,
+  SUBMISSION_TIMESTAMP: 4,
+};
+
+const studentOverviewDataPropType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  numMultipleChoiceCorrect: PropTypes.number.isRequired,
+  numMultipleChoice: PropTypes.number.isRequired,
+  percentCorrect: PropTypes.string.isRequired,
+  submissionTimestamp: PropTypes.string,
```

Comment:
`submissionStatus` and `isSubmitted` is redundant. submissionStatus should be able to include something like `submitted` and then you can check that in your view.

Once we hook this up to the API/redux we'll want to have a valid set of statuses, but for now leaving it as a string is fine.